src/script/common/c_content.cpp

```diff
@@ -879,7 +892,7 @@ void push_content_features(lua_State *L, const ContentFeatures &c)
 	push_ARGB8(L, c.minimap_color);       // I know this is not set-able w/ register_node,
 	lua_setfield(L, -2, "minimap_color"); // but the people need to know!
 #endif
-	lua_pushnumber(L, c.visual_scale);
+	push_v3f(L, c.visual_scale);
```

Comment:
This *will* break backwards compatibility with any mods that read content features and expect `visual_scale` to be a number.

Reply:
I obliged to add the `is_vscale_num` flag in ContentFeatures to specify whether it was initialized as a number or table in Lua.